framework/test-vectors/esdk-test-vector-enumeration.md

```diff
@@ -0,0 +1,59 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# ESDK Test Vector Enumeration
+
+This document performs the [test vector enumeration](test-vector-enumeration.md) process for the ESDK
+to construct the full suite of test vectors
+for the ESDK.
+
+The full suite of test vectors can be constructed
+by [enumerating all input configurations](test-vector-enumeration.md#enumerating-input-configurations) from this spec's input dimensions
+and [evaluating each configuration's expected result](test-vector-enumeration.md#determining-expected-results) from this spec's evaluation rules.
+
+## Input dimensions
```

Comment:
Issue/Blocking: There is a Max Encrypted Data Key setting that MUST be added.
I cannot find that setting in the MPL or ESDK Test Vector Enumeration docs.

Reply:
Added; I put this in the ESDK because I found logic for this in the Python ESDK